doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,1913 @@
+
+You can find all of the installation details in the 
+`Installation <https://www.scipy.org/install.html>`_ section 
+at `SciPy <https://www.scipy.org>`_.
+
+How to import NumPy 
+-------------------
+
+Any time you want to use a package or library in your code, you first need to
+make it accessible. 
+
+In order to start using NumPy and all of the functions available in NumPy,
+you'll need to import it. This can be easily done with this import statement::
+
+  import numpy as np 
+
+(We shorten ``numpy`` to ``np`` in order to save time and also to keep code
+(standardized so that anyone working with your code can easily understand and
+(run it.)
+
+Reading the example code
+------------------------
+
+If you aren't already comfortable with reading tutorials that contain a lot of code, 
+you might not know how to interpret a code block that looks 
+like this::
+
+  >>> a2 = a[np.newaxis, :]
+  >>> a2.shape
+  (1, 6)
+
+If you aren't familiar with this style, it's very easy to understand. 
+If you see ``>>>``, you're looking at **input**, or the code that 
+you would enter. Everything that doesn't have ``>>>`` in front of it 
+is **output**, or the results of running your code.
+
+
+What’s the difference between a Python list and a NumPy array? 
+--------------------------------------------------------------
+  
+NumPy gives you an enormous range of fast and efficient ways of creating arrays
+and manipulating numerical data inside them. While a Python list can contain
+different data types within a single list, all of the elements in a NumPy array
+should be homogenous. The mathematical operations that are meant to be performed
+on arrays would be extremely inefficient if the arrays weren't homogenous. 
+
+**Why use NumPy?**
+
+NumPy arrays are faster and more compact than Python lists. An array consumes
+less memory and is convenient to use. NumPy uses much less memory to store data
+and it provides a mechanism of specifying the data types. This allows the code
+to be optimized even further. 
+
+What is an array? 
+-----------------
+
+An array is a central data structure of the NumPy library. An array is a grid of
+values and it contains information about the raw data, how to locate an element,
+and how to interpret an element. It has a grid of elements that can be indexed
+in `various ways <https://numpy.org/devdocs/user/quickstart.html#indexing-slicing-and-iterating>`_.
+The elements are all of the same type, referred to as the array ``dtype``. 
+
+An array can be indexed by a tuple of nonnegative integers, by booleans, by
+another array, or by integers. The ``rank`` of the array is the number of
+dimensions. The ``shape`` of the array is a tuple of integers giving the size of
+the array along each dimension.
+
+One way we can initialize NumPy arrays is from Python lists, using nested lists
+for two- or higher-dimensional data.. 
+
+For example::
+
+  >>> array = np.array([1, 2, 3, 4, 5, 6])
```

Comment:
would probably stick to the same variable name here
```suggestion
  >>> a = np.array([1, 2, 3, 4, 5, 6])
```

Reply:
Got it!